File: service/src/main/java/hu/bsstudio/robonaut/scores/audience/AudienceScoreService.java
package hu.bsstudio.robonaut.scores.audience;

import hu.bsstudio.robonaut.scores.audience.model.AudienceScoredTeam;
import hu.bsstudio.robonaut.team.model.DetailedTeam;
import reactor.core.publisher.Mono;

public interface AudienceScoreService {
    Mono<DetailedTeam> updateAudienceScore(final AudienceScoredTeam audienceScoredTeam);
}

File: service/src/main/kotlin/hu/bsstudio/robonaut/scores/audience/AudienceScoreService.kt
package hu.bsstudio.robonaut.scores.audience

import hu.bsstudio.robonaut.scores.audience.model.AudienceScoredTeam
import hu.bsstudio.robonaut.team.model.DetailedTeam
import reactor.core.publisher.Mono

interface AudienceScoreService {
    fun updateAudienceScore(audienceScoredTeam: AudienceScoredTeam): Mono<DetailedTeam>
}
